tools/generate-helpers/bootstrap-migration/migration.go.tpl

```diff
@@ -22,4 +22,3 @@ var (
 func init() {
 	migrations.MustRegisterMigration(migration)
 }
-
```

Comment:
I think this might be a change by accident.

Reply:
nope, for some reason -- don't ask me why -- the style checks were complaining about an empty line before EOF here and I fixed it to make the (required!) tests pass.

Reply:
Hmm, for some reasons I did only ever get complains by CI because I didn't have an empty line before EOF.

I'll close this do whatever is required :)